setup.py

```diff
@@ -25,11 +25,6 @@
         'Intended Audience :: System Administrators',
         'Natural Language :: English',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
```

Comment:
We can keep this line in there, since we do have support for versions in Python 3